tests/Stripe/WebhookTest.php

```diff
@@ -79,7 +79,7 @@ public function testNoValidSignatureForPayload()
         $this->expectException(\Stripe\Exception\SignatureVerificationException::class);
         $this->expectExceptionMessage('No signatures found matching the expected signature for payload');
 
-        $sigHeader = $this->generateHeader(['signature' => 'bad_signature']);
+        $sigHeader = $this->generateHeader(['signature' => \hash('sha256', 'bad_signature')]);
```

Comment:
We don't need this to be a valid hash, since it's meant to be a "bad_signature" anyways :)